tools/nnyeah/nnyeah/TransformEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Text;
+
+#nullable enable
+
+namespace nnyeah {
+	public class TransformEventArgs : BaseTransformEventArgs {
+		public TransformEventArgs (string containerName, string methodName, string targetOperand, int addedCount, int removedCount)
+			: base (containerName, methodName, targetOperand)
+		{
+			if (addedCount < 0)
+				throw new ArgumentOutOfRangeException (nameof (addedCount));
+			if (removedCount < 0)
+				throw new ArgumentOutOfRangeException (nameof (removedCount));
```

Comment:
Can we just set the int to be uint? That way we cannot pass a negative value and the compiler will help.

Reply:
I wouldn't have thought of this, but good idea!